cns/NetworkContainerContract.go

```diff
@@ -404,6 +410,10 @@ type PodIpInfo struct {
 	PodIPConfig                     IPSubnet
```

Comment:
we should add comments to the PodIpInfo interface at this point.
we're adding enough fields it's not just a tuple of other ipconfigs anymore - including an enum of non-obvious allowed string values.

Reply:
sure, will update with comments